cpp/modules/deck.gl/core/src/lib/layer.h

```diff
@@ -165,15 +164,17 @@ class Layer : public Component {
   // Called once when layer is no longer matched and state will be discarded: App can destroy WebGL resources here
   virtual void finalizeState();
 
-  // If state has a model, draw it with supplied uniforms
-  virtual void drawState(wgpu::RenderPassEncoder pass);
+  void draw(wgpu::RenderPassEncoder pass);
 
  protected:
   // INTERNAL METHODS
 
   // Default implementation of attribute invalidation, can be redefined
   void invalidateAttribute(const std::string& name = "all", const std::string& diffReason = "");
 
+  // If state has a model, draw it with supplied uniforms
+  virtual void _drawState(wgpu::RenderPassEncoder pass);
```

Comment:
Move back: This needs to be next to the other lifecycle methods, the four virtual methods.